addons/60fps/addon.json

```diff
@@ -4,7 +4,7 @@
   "info": [
     {
       "type": "notice",
```

Comment:
I'd even understand if this was a warning (though the warnings are pretty distracting).